toolium/driver_wrapper.py

```diff
@@ -213,6 +214,20 @@ def connect(self, maximize=True):
                                                                                                 'appium_app_strings'):
             self.app_strings = self.driver.app_strings()
 
+        # Bounds and screen
+        bounds_x = int(self.config.get_optional('Driver', 'bounds_x') or 0)
+        bounds_y = int(self.config.get_optional('Driver', 'bounds_y') or 0)
+
+        monitor_index = int(self.config.get_optional('Driver', 'monitor') or -1)
+        if monitor_index > -1:
+            monitor = screeninfo.get_monitors()[monitor_index]
```

Comment:
Unit tests fails in Travis CI because `get_monitors()` returns `NotImplementedError`. This error should be cached.

Reply:
Done